.teamcity/settings.kts

```diff
@@ -451,7 +451,7 @@ object CheckCodeStyle : BuildType({
 				nvm install
 
 				# Find files to lint
-				if [ "%teamcity.build.branch.is_default%" = "true" ]; then
+				if [ "%teamcity.build.branch.is_default%" = "true" ] || [ "%calypso.run_full_eslint%" = "true" ]; then
```

Comment:
This allow us to toggle "full eslint" (lint all files) or "partial eslint" (lint only changed files) when running a build manually in TeamCity